sdk/examples/c2pa.toml

```diff
@@ -66,9 +65,8 @@ name = "My Service"
 # A human readable string of the product's version.
 version = "1.0.0"
 # The operating system the claim generator is running on.
-operating_system.name = "macOS"
-# Or if the name isn't specified, it can be inferred automatically.
-operating_system.infer = true
+# Or specify "auto" to infer the operating system automatically.
+operating_system = "macOS"
```

Comment:
What are the valid values here? The spec isn't very helpful but we should be better. Are we following any known naming convention for this?

Reply:
Maybe this is something we should add to the `ClaimGeneratorInfoOperatingSystem` settings, alongside `Auto`?

Reply:
I'd recommend something like the compiler triplet convention, but that's not just an operating system.

x86_64-unknown-linux-gnu
aarch64-apple-darwin
x86_64-pc-windows-msvc
arm-linux-gnueabihf